Doc/conf.py

```diff
@@ -30,20 +31,20 @@
 
 # Skip if downstream redistributors haven't installed them
 try:
-    import notfound.extension
+    import notfound.extension  # NoQA: F401
```

Comment:
micro-nit: I more usually see these suppression comments as all-lowercase, e.g.

```suggestion
    import notfound.extension  # noqa: F401
```

Reply:
I prefer "NoQA" as it reads (to me) more like "no quality assurance" -- I remember being perplexed as to what this random non-English word 'noqa' meant when I first started reading source files. Perhaps there should be a formatting option for the capitalisation of NoQA comments ;)

Reply:
Haha, it's been suggested on our issue tracker IIRC...

Reply:
I don't think I've ever seen "NoQA" used before.

In the top 8k PyPI projects:

* `# noqa`: Found 494470 matching lines in 2766 projects
* `# NOQA`: Found 3204 matching lines in 252 projects
* `# NoQA`: Found 146 matching lines in 17 projects
  * A lot of these are Sphinx ;)

So NoQA is only 0.03% of examples.

Ruff's documentation has "noqa": https://docs.astral.sh/ruff/linter/#error-suppression
Flake8 is mostly noqa"" with some "NOQA": https://flake8.pycqa.org/en/latest/search.html?q=noqa&check_keywords=yes&area=default

I think we should stick with the most common "noqa", it's the more familiar choice.

Reply:
> A lot of these are Sphinx ;)

Never!

---

I shall update with the wisdom of the crowd.